slick/src/test/scala/geotrellis/slick/PostgisSpec.scala

```diff
@@ -205,45 +209,41 @@ class PostgisSpec extends FlatSpec with ShouldMatchers with TestDatabase {
   val OptCity = TableQuery[OptCityRow]
 
   it should "be able to handle optional fields" in {
-    db withSession { implicit s =>
-      try { OptCity.ddl.drop } catch { case e: Throwable => }
+      try { db.run(OptCity.schema.drop).futureValue } catch { case e: Throwable => }
 
-      OptCity.ddl.create
+      db.run(OptCity.schema.create).futureValue
 
       val cities = Seq((0, "washington",Some(pt(-77.02,38.53))),
         (0, "london", None),
         (0, "paris", Some(pt(2.3470,48.8742))))
 
-      OptCity ++= cities
-
+      db.run(OptCity ++= cities).futureValue
+     //For some reason isEmpty does not work
```

Comment:
This comment only makes sense in the context of the diff, would be confusing to someone arriving at it without knowing were `isEmpty` was called beforehand; remove.